RecoLocalCalo/HcalRecAlgos/src/HybridMinimizer.cc

```diff
@@ -320,17 +320,19 @@ bool HybridMinimizer::Minimize() {
    int strategyLevel = Strategy(); 
    fMinuitFCN->SetErrorDef(ErrorDef() );
 
+   /*
```

Comment:
All the commented out parts of code in this HybridMinimizer.cc should be better removed, or alternatively some additional line of comment should be added to explain why they must remain there

Reply:
In this file there are large chunks of code commented out, as they are apparently not needed: they should be better removed. Nonetheless, I will give my "+1" to this PR just in case it can still be allowed (as "bug fix") in 10_0_X: in that case ```HybridMinimizer.cc``` will be adjusted in a forthcoming pull request. If this PR does not enter 10_0_X, or (even better) if it can be adjusted quickly now, I will remove the "+1" waiting for it